shims/hdp26/kar/pom.xml

```diff
@@ -0,0 +1,58 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <parent>
+    <artifactId>pentaho-hadoop-shims-hdp26-reactor</artifactId>
+    <groupId>org.pentaho</groupId>
+    <version>9.0.0.0-SNAPSHOT</version>
+  </parent>
+  <modelVersion>4.0.0</modelVersion>
+
+  <artifactId>pentaho-hadoop-shims-hdp26-kar</artifactId>
+
+  <version>9.0.0.0-SNAPSHOT</version>
+  <packaging>kar</packaging>
+  <dependencies>
+    <dependency>
+      <groupId>org.pentaho</groupId>
+      <artifactId>pentaho-hadoop-shims-common-dependencies-V1</artifactId>
+      <version>9.0.0.0-SNAPSHOT</version>
+    </dependency>
+    <dependency>
+      <groupId>org.pentaho</groupId>
+      <artifactId>pentaho-hadoop-shims-common-fragment-V1</artifactId>
+      <version>9.0.0.0-SNAPSHOT</version>
+    </dependency>
+    <dependency>
+      <groupId>org.pentaho</groupId>
+      <artifactId>pentaho-hadoop-shims-hdp26-driver</artifactId>
+      <version>9.0.0.0-SNAPSHOT</version>
+    </dependency>
+  </dependencies>
+  <build>
+    <plugins>
+      <plugin>
+        <groupId>org.apache.karaf.tooling</groupId>
+        <artifactId>karaf-maven-plugin</artifactId>
+        <version>${karaf-maven-plugin.version}</version>
+        <extensions>true</extensions>
```

Comment:
Why not use the profile for features from the parent pom?
https://github.com/pentaho/maven-parent-poms/blob/master/pom.xml#L1669
I believe that with that profile you should also be able to produce a kar as well without issues.